### ISSUE TITLE
BGP routes are not getting cleared in the ASIC tables under stress tests

### ISSUE THREAD

**Issue by rubensfig** [bisdn/basebox#376]
## Expected Behavior

When adding multiple (>1000) routes over BGP, the installed routes should all disappear after the remote peer disconnects. 

```
Before BGP peer is started
172.16.111.1 dev enp0 proto dhcp scope link src 172.16.111.15 metric 10
accton-as4610:/home/basebox# ip r | wc -l
7
accton-as4610:/home/basebox# client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 7.
```

```
After BGP peer is started
accton-as4610:/home/basebox# ip r | wc -l
2048
accton-as4610:/home/basebox# client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 2048.
```

```
After BGP peer is stopped
accton-as4610:/home/basebox# ip r | wc -l
7
accton-as4610:/home/basebox# client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 7.
```

## Actual Behavior

The routes disappear in the kernel, but are maintained in the ASIC, leading to misconfiguration.

## Steps to Reproduce the Problem

Setup BGP on a BISDN Linux and a server. The server announces multiple (>1000) routes, wait for BGP to converge, and stop the daemon after.

```
Before BGP peer is started
accton-as4610:/home/basebox# client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 6.
--  etherType = 0x0800 vrf:mask = 0x0000:0x0000 dstIp4 = 10.0.0.0/255.255.255.0 dstIp6 = ::/:: | GoTo = 60 (ACL Policy) outPort = CONTROLLER (Reserved)  | priority = 2 hard_time = 0 idle_time = 0 cookie = 16
--  etherType = 0x0800 vrf:mask = 0x0000:0x0000 dstIp4 = 10.0.0.1/255.255.255.255 dstIp6 = ::/:: | GoTo = 60 (ACL Policy) groupId = 0x20000001 | priority = 3 hard_time = 0 idle_time = 0 cookie = 19

accton-as4610:/home/basebox# ip r
10.0.2.0/24 dev port2 proto kernel scope link src 10.0.2.1
```

```
After BGP peer is started, and BGP convergence achieved.
accton-as4610:/home/basebox# ip r | wc -l
2048

client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 2045.
```

```
After BGP is stopped,
accton-as4610:/home/basebox# client_flowtable_dump 30 | head
Table ID 30 (Unicast Routing):   Retrieving all entries. Max entries = 32768, Current entries = 563.

accton-as4610:/home/basebox# ip r | wc -l
7
```

This shows that not all routes are correctly learned in the first place, and deletion does not accurately work either.

## Specifications
  - Version: BISDN Linux v3.7.2
  - Platform: accton-as4610
  - Subsystem: l3

**Comment by zbroz** [comment]
I believe the problem lies in  net.core.rmem_max and rmem_default being too low. Basebox' cnetlink module uses these values to determine netlink socket cache size. I've been using nlmon to sniff packets and kernel does send all the route notifications correctly, so the problem has to be reception side. After setting 'rmem_default' and 'rmem_max' to 462144, I was able to correctly install 2000 routes.

You definitely need to increase default values.

I am unable to test whether this fixes also route deletion, so please test it, and give us some feedback.

**Comment by hilmarm** [comment]
This issue was fixed in [bisdn/meta-switch#62 Set max read buffer size](https://github.com/bisdn/meta-switch/pull/62)
